projects/plugins/jetpack/extensions/blocks/sharing-button/sharing-sources-block.php

```diff
@@ -0,0 +1,1397 @@
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
```

Comment:
Since we're starting fresh, maybe we could rename that file to match the class?

Reply:
There's a problem that that file contains lots of classes inside. Or we should need at least one?

Reply:
Maybe this is an opportunity to organize this with one class per file?

Reply:
Oh, I might was getting that wrong. I will try to address this.

Reply:
> Maybe this is an opportunity to organize this with one class per file?

Well, they are quite similar. I was actually wondering if, when finalized we could find a way to combine these into one class. We would need to see first how much logic there would differ. 
As of now, there's only 1 function actually, while before there were a lot of frontend stuff.